lib/scala/logging-config/src/main/java/org/enso/logger/config/Appender.java

```diff
@@ -0,0 +1,77 @@
+package org.enso.logger.config;
+
+import com.typesafe.config.Config;
+import java.nio.file.Path;
+import org.enso.logger.LoggerSetup;
+import org.slf4j.event.Level;
+
+/**
+ * Base class for all appenders supported by Enso's logging configuration. Appenders determine what
+ * to do with the recorded log events
+ */
+public abstract class Appender {
```

Comment:
Given that the contract of `LoggerSetup` limits the kinds of appenders that can be created, I think it would make sense to make `Appender` sealed, as there are no other implementations expected.

```suggestion
public sealed abstract class Appender permits ConsoleAppender, FileAppender, ... {
```

Reply:
Fancy, I only see now that we started using this